yt_dlp/extractor/showuptv.py

```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
+from .common import InfoExtractor
+from datetime import datetime
+
+
+class ShowupTvIE(InfoExtractor):
+    IE_NAME = 'showup.tv'
+    _VALID_URL = r"https?://(?:www\.)?showup\.tv/(?P<id>.+)"
+
+    _TESTS = [{
+        "url": "https://showup.tv/username",
+        "info_dict": {
+            "id": "username",
+            "ext": "mp4",
+            'is_live': True,
+        },
+        "skip": "Website content changes dynamically, depending which users are online and available for download/streaming",
+    }, {
+        "url": "https://showup.tv/username",
+        "only_matching": True,
+    }]
+
+    def _real_extract(self, url):
+        user_id = self._match_id(url)
+
+        self._set_cookie("showup.tv", "accept_rules", "true")
+        webpage = self._download_webpage(url, user_id)
+
+        stream_id = self._html_search_regex(r"player\.streamID = '(.*?)'", webpage, "stream_id")
+        server_url = self._html_search_regex(r"player\.transcoderAddr = '(.*?)'", webpage, "server_url")
+
+        rtmp_url = "rtmp://%s:1935/webrtc/%s_aac" % (server_url, stream_id)
+
+        dt_string = datetime.now().strftime("%d-%m-%Y_%H:%M:%S")
+        title = f"showup.tv [{user_id}] {dt_string} {stream_id}"
+
+        return {
+            "id": user_id,
```

Comment:
I think this should be stream_id

Reply:
stream_id is 32 chars string, here is is for user which is streaming

Reply:
so? stream_id is the UNIQUE id of the video, no?
`user_id` should be the `uploader_id`